spec/rails_helper.rb

```diff
@@ -119,3 +118,11 @@
     }
   }
   })
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'spec/fixtures/vcr_cassettes'
+  config.hook_into :webmock
+  config.filter_sensitive_data('SECRET_TMDB_API_KEY') { ENV['TMDB_API_KEY'] }
+  config.default_cassette_options = { re_record_interval: 1.day }
+  config.configure_rspec_metadata!
```

Comment:
configure_rspec_metadata lets us use test names as the name of the fixtures files.